core/lib/spree/testing_support/common_rake.rb

```
@@ -16,10 +16,10 @@
     Spree::InstallGenerator.start ["--lib_name=#{ENV['LIB_NAME']}", "--auto-accept", "--migrate=false", "--seed=false", "--sample=false", "--quiet", "--user_class=#{args[:user_class]}"]
 
     puts "Setting up dummy database..."
-    system("bundle exec rake db:drop db:create db:migrate > #{File::NULL}")
+    system("bundle exec rake db:drop db:create db:migrate > #{File::NULL} 2>&1")
```

I'd argue we want all the output. Especially as this command fails silently (that should be fixed separately, ideally in this branch). And if it fails it should be able to print its reason, reducing the debug cycles from having to repro the failure.